compiler-rt/test/builtins/Unit/trunctfxf2_test.c

```diff
@@ -1,10 +1,10 @@
-// RUN: %clang_builtins %s %librt -o %t && %run %t
+// RUN: %clang_builtins %s %librt -fforce-enable-int128 -o %t && %run %t
```

Comment:
Will adding this flag unconditionally break other/older compilers? Maybe this needs to be a lit substitution?